src/OSPSuite.UI/Extensions/GridGroupRowFormatsExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OSPSuite.Assets;
+using OSPSuite.Core.Chart;
+
+namespace OSPSuite.UI.Extensions
+{
+   public static class GridGroupRowFormatsExtensions
+   {
+      public static string GetFormatString(this GridGroupRowFormats formatName)
+      {
+         switch (formatName)
+         {
+            case GridGroupRowFormats.HideColumnName:
+               return "[#image]{1} {2}";
+            default:
+               return Captions.Chart.GroupRowFormat.GridGroupingRowDefaultFormat; 
```

Comment:
sorry I was not clear. Create an ENUM Values for this like you have with the other one
also the enum 

   public enum GridGroupRowFormats
   {
      HideColumnName
   }

needs to be moved here as well. 
No need to create a constant for `Captions.Chart.GroupRowFormat.GridGroupingRowDefaultFormat` This has nothing to do with chart. Just defined it here in line, it's fine as you do for the HideColumnNam,e. Just create a a default in ten enum that's all

```
   public enum GridGroupRowFormats
   {
      HideColumnName
      Default


   }

Reply:
OK, now I see how you mean. Done.